protostar-rust/tests/data/simple_test/tests/test_simple.cairo

```diff
@@ -8,6 +8,11 @@ fn test_simple2() {
     assert(3 == 3, 'simple check');
 }
 
+#[test]
+fn test_two() {
+    assert(2 == 2, '2 == 2');
+}
+
```

Comment:
I don't see a point in this test, what does it bring?

Reply:
It is used in one of the e2e tests.